docs/learn/learn-account-generation.md

```diff
@@ -173,7 +173,7 @@ feasible in your situation.
 
 ### Go to Polkadot-JS Apps
 
-Navigate to [Polkadot-JS Apps](https://polkadot.js.org/apps) and click on "Accounts" underneath the
+Navigate to [Polkadot-JS Apps](https://dotapps.io/) and click on "Accounts" underneath the
```

Comment:
Navigate to [Polkadot-JS Apps](https://polkadot.js.org/apps) (or [dotapps.io](https://dotapps.io/)) and click on "Accounts" underneath the